src/_pytest/fixtures.py

```diff
@@ -986,7 +1008,15 @@ def __call__(self, function):
         return function
 
 
-def fixture(scope="function", params=None, autouse=False, ids=None, name=None):
+def fixture(
+    callable_or_scope=None,
+    *,
```

Comment:
Technically, changing all of these to keyword-only arguments is a breaking change. Not sure if there's anyone passing these as positional, though.

Reply:
Good catch. Hmmm it is really tricky to deprecate this though... I think we need to change this signature to `*args, **kwargs` and do a lot of juggling. 😬 

Any opinions?

Reply:
Right, someone might rely on the positional args. I'll try with the `*args, **kwargs` approach. I see no other option to keep the backward compatibility.

Reply:
OK, I think I got this. Could you take a look if I didn't overcomplicate it?